samples/bluetooth/mesh/sensor_client/src/model_handler.c

```diff
@@ -93,10 +95,11 @@ static void get_data(struct k_work *work)
 	int err;
 
 	/* Only one message can be published at a time. Swap sensor after each timeout. */
-	switch (sensor_idx++ % 3) {
+	switch (sensor_idx++ % 4) {
```

Comment:
This has the same situation with wrap-around like I mentioned in the previous PR. Should be moduloed at assignment.